docs/guides/introduction.md

```diff
@@ -22,47 +14,54 @@ ZKPool aims to provide the following capabilities:
 
 Initially, ZKPool will assume the risk and stake on behalf of all the provers connected to the pool. The pool is open for registered provers to connect to.
```

Comment:
“stake” is not appropriate here for a5 anymore?

Reply:
should be OK. Bond is a kind of stake.

Reply:
done